scipy/stats/tests/test_morestats.py

```diff
@@ -929,51 +934,51 @@ def test_pratt(self):
         y = [1, 2, 3, 5]
         with suppress_warnings() as sup:
             sup.filter(UserWarning, message="Sample size too small")
-            res = stats.wilcoxon(x, y, zero_method="pratt")
+            res = stats.wilcoxon(x, y, zero_method="pratt", method="approx")
         assert_allclose(res, (0.0, 0.31731050786291415))
 
     def test_wilcoxon_arg_type(self):
         # Should be able to accept list as arguments.
         # Address issue 6070.
         arr = [1, 2, 3, 0, -1, 3, 1, 2, 1, 1, 2]
 
-        _ = stats.wilcoxon(arr, zero_method="pratt")
-        _ = stats.wilcoxon(arr, zero_method="zsplit")
-        _ = stats.wilcoxon(arr, zero_method="wilcox")
+        _ = stats.wilcoxon(arr, zero_method="pratt", method="approx")
+        _ = stats.wilcoxon(arr, zero_method="zsplit", method="approx")
+        _ = stats.wilcoxon(arr, zero_method="wilcox", method="approx")
```

Comment:
Could be worth parametrizing here for the three similar calls, but that could be done in a follow-up.